02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,115 @@
+        <img src="data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBxISEhUSExIVFRUXFRUVFRUVFRUVFhUVFRUXFxUVFxUYHSggGBolHRcVITEiJSkrLi4uFx8zODMsNygtLisBCgoKDg0OGhAQGy0mHyUtLS0tLS0tLS0vLS0tLS8tLS0tLS0tLS0tLS8tLS0tLy0tLS0tKy0tLS0tLS0tLS0tLf/AABEIAMIBAwMBIgACEQEDEQH/xAAbAAABBQEBAAAAAAAAAAAAAAAAAQIDBAUGB//EAD0QAAEDAgQDBgQFAgUEAwAAAAEAAhEDIQQSMUEFUWEGEyJxgZEyobHwFCNC0eFSwTNicoKSBxU08UOywv/EABoBAQEBAQEBAQAAAAAAAAAAAAABAgMEBQb/xAArEQACAgICAgAFAgcAAAAAAAAAAQIRAyESMQRBEyJRYfAycRSBkaGx4fH/2gAMAwEAAhEDEQA/APDkIQgBCE4BAJCUBODVI1iEsjAUrWJct1b
```

Comment:
Wow you are really hitting the ground running! ![Great Job](https://media.giphy.com/media/wQNbW62D5XSkU/giphy.gif)